drivers/usb/device/usb_dc_mcux.c

```diff
@@ -95,6 +95,8 @@ BUILD_ASSERT(NUM_INSTS <= 1, "Only one USB device supported");
 #elif DT_NODE_HAS_STATUS(DT_NODELABEL(usb2), okay)
 #define CONTROLLER_ID kUSB_ControllerEhci1
 #endif /* IMX RT */
+#elif defined(CONFIG_SOC_SERIES_RW6XX)
+#define CONTROLLER_ID kUSB_ControllerEhci0
```

Comment:
Can we avoid adding this by using the nod label `usb1`?

Reply:
We can't, no- take a look at how the code above depends on `#elif defined(CONFIG_SOC_SERIES_IMXRT11XX) || defined(CONFIG_SOC_SERIES_IMXRT10XX)`. I discussed this with @decsny here: https://github.com/zephyrproject-rtos/zephyr/pull/63999#discussion_r1383676505. I still am of the opinion we should not be introducing SOC specific Kconfigs into a driver unless it is absolutely required, but that is outside the scope of this PR.

Reply:
Makes sense. No point spending time restructuring this driver since we are moving to a new driver.